py/server/tests/test_pyfunc_return_java_values.py

```diff
@@ -271,6 +271,27 @@ def f1(col) -> Optional[List[int]]:
         self.assertEqual(t.columns[0].data_type, dtypes.long_array)
         self.assertEqual(t.to_string().count("null"), 5)
 
+    def test_np_ufunc(self):
+        # no vectorization and no type inference
+        npsin = np.sin
+        t = empty_table(10).update(["X1 = npsin(i)"])
+        self.assertEqual(t.columns[0].data_type, dtypes.PyObject)
+        t2 = t.update("X2 = X1.getDoubleValue()")
```

Comment:
Does a double cast work?

Reply:
No, the column is really of PyObject type.